presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -49,17 +49,16 @@ will create a catalog named ``sales`` using the configured connector.
 HDFS Configuration
 ^^^^^^^^^^^^^^^^^^
 
-Presto configures the HDFS client automatically for most setups and
-does not require any configuration files. In some rare cases, such
-as when using federated HDFS, it may be necessary to specify additional
-HDFS client options in order to access your HDFS cluster. To do so, add
-the ``hive.config.resources`` property to reference your HDFS config files:
+For basic setups, Presto configures the HDFS client automatically and
+does not require any configuration files. In some cases, such as when using
+federated HDFS or NameNode high availability, it is necessary to specify
+additional HDFS client options in order to access your HDFS cluster. To do so,
+add the ``hive.config.resources`` property to reference your HDFS config files:
 
 .. code-block:: none
 
     hive.config.resources=/etc/hadoop/conf/core-site.xml,/etc/hadoop/conf/hdfs-site.xml
 
-Only specify additional configuration files if absolutely necessary.
```

Comment:
This line should stay. Adding additional config files could cause problems, so it is better not to do it unless necessary.